cmd/generator/main.go

```diff
@@ -197,7 +199,8 @@ func getReleasePlugins(ctx context.Context, client *github.Client, repositoryNam
 		}
 		signatures, err := downloadSignatures(signatureAssets)
 		if err != nil {
-			return nil, errors.Wrapf(err, "failed to download signatures for release %s", releaseName)
+			logger.Error(errors.Wrapf(err, "failed to download signatures for release %s", releaseName))
```

Comment:
As above, why aren't we just erroring out? (Won't repeat this comment everywhere, but of course it applies below.)

Reply:
Looks like most of my errors were coming from attempting to parse old style bundles. Ignoring those and erroring out on everything now.